cue_file

```diff
@@ -186,22 +185,13 @@ def analyse(
                 break
         start_next_time = max(start_next_time, cue_out_time - start_next_time)
 
-    # Now that we know where to start the next song, calculate Liquidsoap's
-    # cross duration from it, allowing for an extra 0.1s of overlap -- no, reverse
-    # (a value of 0.0 is invalid in Liquidsoap)
-    cross_duration = cue_out_time - start_next_time
-
-    # We now also return start_next_time
-    
     # return a dict
     return ({
         "duration": duration,
-        "liq_duration": cue_duration,
```

Comment:
This is remove to avoid potentially confusing liquidsoap. You can bring it back as prefixed meta (i.e. `liq_cue_file_duration`) if you want.